packages/venia-concept/src/components/CreateAccount/createAccount.js

```diff
@@ -174,14 +192,15 @@ class CreateAccount extends Component {
     }
 
     onCreateAccount = () => {
+        const { getFieldValue } = this;
```

Comment:
You can just use `formState` much how you're passing `formState` into `handleFormState` here. I'm not sure why the whole form is duplicated into the component `state`.